R/database.R

```diff
@@ -1279,6 +1274,16 @@ get_manta_sv = function(min_vaf = 0.1,
 
     all_sv = suppressMessages(read_tsv(sv_file, col_types = "cnncnncnccccnnnnccc", col_names = cnames))
```

Comment:
To address the issue I raised above, can you just replace this line with a call to get_combined_sv()
e.g.
all_sv = get_combined_sv()

Reply:
This function has now been updated to use `get_combined_sv` (replacing, `get_merged_results`).

Reply:
When testing, I discovered that the output from `get_combined_sv()`, does not match the output from the previous function (`get_merged_results()`). The columns are quite different, meaning that this code from `get_manta_sv` fails; `all_sv = rbind(all_sv, missing_sv)`. 

The question is, should we revert and still `call get_merged_results` in `get_manta_sv`, or should the return for `get_manta_by_sample` be modified so that it matches what gets returned with `get_combined_sv`?

Reply:
I would go with option 2!